tests/Unit/HtmlProcessor/HtmlPrunerTest.php

```diff
@@ -606,7 +629,7 @@ private static function assertContainsAll(array $needles, string $haystack)
      *
      * @throws AssertionFailedError
      */
-    private static function assertSubstringCount(int $expectedCount, string $haystack, string $needle)
+    private function assertSubstringCount(int $expectedCount, string $haystack, string $needle)
```

Comment:
Why make this non-static?

In the whole of PHPUnit, the assert methods are all `static`.  So when we introduce 'extension' methods, we retain consistency my making them `static` too.

Reply:
They oddly didn't come up with `NonStaticSelfCall`.

Reply:
> They oddly didn't come up with `NonStaticSelfCall`.

See #792.

Reply:
reverted in rebase referenced https://github.com/MyIntervals/emogrifier/pull/778#issuecomment-538681924